### PR TITLE
Fix watsonx service url credential field

### LINKED ISSUE #3608: [BUG] IBM WatsonX serviceUrl credential
**Describe the bug**
Currently, the `serviceUrl` credential field for IBM WatsonX is set to be mandatory.

**Expected behavior**
The input field should be optional.

**Additional context**
https://github.com/FlowiseAI/Flowise/blob/main/packages/components/credentials/IBMWatsonx.credential.ts

### PR DESCRIPTION
**Pull request by eduardconstantin**
Fixes #3608

I checked the type for watsonx auth fields and it seems the service url is mandatory, but I tested without using it in the credential form and the model still worked.

### CODE REVIEW

File: packages/components/credentials/IBMWatsonx.credential.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ class IBMWatsonxCredential implements INodeCredential {
     inputs: INodeParams[]
 
     constructor() {
-        this.label = 'IBM Watsonx'
+        this.label = 'IBM WatsonX'
```
[the remark this answers is not in the document]
Comment (eduardconstantin) [reply packages/components/credentials/IBMWatsonx.credential.ts:11]: Yes, my mistake, I saw now that they use the x in lowercase

**Comment by HenryHengZJ** [comment]
Actually we need service URL, without it, it says Not Found:

image

Will be closing this PR for now

**Comment by HenryHengZJ** [comment]
With service URL: 
image

**Comment by eduardconstantin** [comment]
But how does it work for me?
I just checked the credentials form and I have nothing added to service url

**Pull request closed without merging**